.github/workflows/build.yml

```diff
@@ -135,6 +135,7 @@ jobs:
       run: |
         rm -rf /Applications/Xcode.app
         ln -s /Applications/Xcode_11.7.app /Applications/Xcode.app
+        ln -s /Applications/Xcode_11.7.app /Applications/Xcode-11.7.app
```

Comment:
This looks a bit strange - is this because the github actions runners have it in a different place from the default? i.e. with a `_` instead of an `-` and we have stuff hard coded to use the location with a `-`?

Reply:
Yes, the runner has it in /Applications/Xcode_11.7.app while the build expects it to be in /Applications/Xcode-11.7.app

Reply:
Presumably one of those is the default location and the discrepancy isn't just because we've changed the default in which case we'd be able to change it)?